src/extensions/extensions-store.ts

```diff
@@ -43,16 +43,6 @@ export class ExtensionsStore extends BaseStore<LensExtensionsStoreModel> {
     await extensionLoader.whenLoaded;
     await this.whenLoaded;
 
-    // apply state on changes from store
```

Comment:
Why is these lines removed? Removing this reaction seems to break enabling/disabling extensions from UI

Reply:
Removing these lines makes it possible to set isEnabled = true in the reaction. I didn't see any bugs from this change at least in my testing..

It would be nice for someone to explain the logic of why we need these lines.

Reply:
Okay, this does break the disabling to an extent. I'll change the PR yet again.

Reply:
PR fixed, enable and disable should work now.